http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/PlainBookResource.java

```diff
@@ -27,6 +33,21 @@ public Uni<Book> getByQueryMap(@QueryParam("param") Map<String, String> params)
                         params.get("author")));
     }
 
+    @GET
+    @Path("/rest-query")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Uni<List<Book>> getRestQuery(@RestQuery Integer id1, @RestQuery Integer id2,
```

Comment:
Parameter names are inconsistent("id2" vs "thirdBookId"). I presume, you wanted to show, that they are coming from different maps, but I recommend to either write parameter names separately(something like `@RestQuery("thirdBookId") BookIdWrapper thirdId`) or give them more concise names(something like "firstPlainId", "secondObjectId" and so on)

Reply:
True, fixed, thanks.